core/templates/dev/head/pages/community-dashboard-page/modal-templates/question-suggestion-review.directive.html

```diff
@@ -17,14 +17,47 @@ <h4 class="author-header">Submitted by <[authorName]></h4>
       </question-editor>
     </div>
   </section>
-  <div ng-show="reviewable" class="oppia-suggestion-review-message">
-    Review message (required if rejecting):
-    <input class="protractor-test-suggestion-review-message" type="text" ng-model="reviewMessage" style="width: 100%">
-  </div>
-  <div ng-show="reviewable" class="oppia-suggestion-review-message">
-    Brief Description of Changes (required if accepting):
-    <input class="protractor-test-suggestion-commit-message" type="text" ng-model="commitMessage" style="width: 100%">
-  </div>
+  <section class="oppia-reviewer-actions">
+    <div class="oppia-suggestion-review-message" ng-show="reviewable">
+      <div class="oppia-difficulty-message">Please select an appropriate difficulty for the question.</div>
+      <md-radio-group ng-model="difficulty">
+        <md-radio-button value="0.3">
+          <strong> Easy </strong>
+          <angular-html-bind html-data="skillRubrics[0].getExplanation()">
+          </angular-html-bind>
+          <div ng-if="skillRubrics[0].getExplanation() === ''">
+            <span class="oppia-placeholder"> This rubric has not yet been specified </span>
+          </div>
+        </md-radio-button>
+
+        <md-radio-button value="0.6">
```

Comment:
I see this being repeated multiple times can we use ng-repeat and bind it with skillRubrics?